code/src/components/Header.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+export const Header = () => {
+  return (
+    <header>
+      <div className="logo">
+        <img className="logoImg" src="/cinema_city.png" alt="" />
+      </div>
+      <ul className="navlinkWrapper">
+        <li className="link">
+          <NavLink to="/">Home</NavLink>
```

Comment:
Hey!, maybe change this to what it actually is (and says on the page when you open that part) - popular movies?